napari/_vispy/vendored/volume.py

```diff
@@ -427,6 +427,13 @@ class VolumeVisual(Visual):
 
     _interpolation_names = ['linear', 'nearest']
 
+    _complex_modes = {
```

Comment:
I liked the subclassing approach you took with `ImageVisual` to make `ComplexImageVisual` - just curious why you didn't go with that approach here? I think it might make things easier for us to have less vendored vispy code after their next release etc.